lib/runtime/reducers.js

```diff
@@ -288,14 +289,21 @@ reducers.sigma = {
                 } else if (n === 1) {
                     return 0;
                 } else {
-                    return math.sqrt(ops.mul(1/(n-1), ops.sub(ssum, ops.div(ops.mul(sum, sum), n))));
+                    return math.sqrt(ops.div(ops.sub(ssum, ops.div(ops.mul(sum, sum), n)), n));
                 }
             },
             update: function(pt) {
                 var v = pt[fld];
                 if (v !== undefined) {
-                    sum = ops.add(sum, v);
-                    ssum = ops.add(ssum, ops.mul(v, v));
+                    // Reduce error using the assumed mean method by shifting
+                    // the data based on the first value as suggested by:
+                    // https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance
```

Comment:
Can we move this comment to just above `reducers.sigma = {` as part of a larger comment that describes the whole algorithm we use including that Bessel thing?